src/main/kotlin/org/rust/lang/core/RsPsiPattern.kt

```diff
@@ -169,12 +176,17 @@ object RsPsiPattern {
             .withSuperParent(2, RsOuterAttributeOwner::class.java)
             .with("nonStdAttributeCondition") { e -> e.name !in STD_ATTRIBUTES }
 
+    val lintAttributeMetaItem: PsiElementPattern.Capture<RsMetaItem> =
```

Comment:
JFYI there is `org.rust.lang.core.completion.RsPsiPatternTest` to check that patterns works expected.
We usually don't use unit test and use integration instead but since patterns can be quite complex, these tests can be helpful